packages/orchestration/src/types.d.ts

```diff
@@ -1,8 +1,15 @@
 import type { RouterProtocol } from '@agoric/network/src/router';
 
 export type ConnectionId = `connection-${number}`;
+export type ChannelId = `channel-${number}`;
+export type DestinationPort = 'icahost' | 'icqhost';
+export type SourcePortPrefix = 'icacontroller' | 'icqcontroller';
+export type SourcePort = `${SourcePortPrefix}-${number}`;
+// TODO improve. sometimes expressed uppercase, others lowercase
```

Comment:
+1, I don't get it

Reply:
I removed this type (ChannelOrdering) in favor of the now exported `@typedef {'ORDERED' | 'UNORDERED'} IBCChannelOrdering` from `@agoric/vats`. It seems like the lowercase variant is used when creating an address string, and the uppercase variant in packets.  There is no type checking against address strings in this PR, but we could maybe add something like `type AddressChannelOrdering =  Lowercase<IBCChannelOrdering>`

Reply:
> It seems like the lowercase variant is used when creating an address string, and the uppercase variant in packets.

Agreed, but I don't understand why